Sources/AppStoreConnectCLI/Services/Operations/ListBetaGroupsOperation.swift

```diff
@@ -33,7 +34,10 @@ struct ListBetaGroupsOperation: APIOperation {
     }
 
     func execute(with requestor: EndpointRequestor) -> AnyPublisher<Output, Swift.Error> {
-        let filters = options.appIds.isEmpty ? [] : [ListBetaGroups.Filter.app(options.appIds)]
+        var filters: [ListBetaGroups.Filter] = []
+        filters += options.appIds.isEmpty ? [] : [.app(options.appIds)]
+        filters += options.names.isEmpty ? [] : [.name(options.names)]
+
```

Comment:
not relevant to this PR, but after adding name as a filter I think the `ListBetaGroupOperation` should throw some Error if there are no groups in response. 🤔

Reply:
🤔 I'm not sure about that. In the list operation we're just filtering down the list. If the list happens to be empty after filtering that's okay I think.
The user will instinctively know to ask for the list with less filtering.

Reply:
It will show a list with one empty line on it. I am not sure coz we seem did some error thrown in other listing commands when `data.count == 0` 🤔

Reply:
Oh 🤔 we should be consistent on this. I think showing empty data is okay and doesn't need an error?

What do you think?